File: Tutorial/NDOTestApp/NDOTravelExpenses-Step 6 VB/BusinessClasses/Address.vb
Imports NDO
Imports System


<NDOPersistent()> _
Public Class Address
    ' Properties
    Public Property City() As String
        Get
            Return _city
        End Get
        Set(ByVal value As String)
            _city = value
        End Set
    End Property

    Public Property CountryCode() As String
        Get
            Return _countryCode
        End Get
        Set(ByVal value As String)
            _countryCode = value
        End Set
    End Property

    Public Property Street() As String
        Get
            Return _street
        End Get
        Set(ByVal value As String)
            _street = value
        End Set
    End Property

    Public Property Zip() As String
        Get
            Return _zip
        End Get
        Set(ByVal value As String)
            _zip = value
        End Set
    End Property


    ' Fields
    Private _city As String
    Private _countryCode As String
    Private _street As String
    Private _zip As String
End Class

File: Tutorial/NDOTestApp/NDOTravelExpenses-Step 3/BusinessClasses/Address.cs
using System;
using NDO;
namespace BusinessClasses
{
    [NDOPersistent]
    public class Address
    {
        private string countryCode;
        private string zip;
        private string street;
        private string city;
        public Address()
        {
        }
        public string CountryCode
        {
            get { return countryCode; }
            set { countryCode = value; }
        }
        public string Zip
        {
            get { return zip; }
            set { zip = value; }
        }
        public string Street
        {
            get { return street; }
            set { street = value; }
        }
        public string City
        {
            get { return city; }
            set { city = value; }
        }
    }
}
